### DISCUSSION TITLE
Used Copilot for Individuals Free Trial, Can't subscribe

### DISCUSSION THREAD

**Discussion by datalore92** [community/community#69129] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I've been using the Github Copilot free trial for the past month and decided that I like it and want to keep using it.
My free trial ends today, so I figured I should go to my Github Copilot billing settings on my Github account and subscribe to it monthly.  However, there is no option that allows me to subscribe monthly for Copilot.  
I click on my account, I go to Settings, I go to Billing and Plans, I go to Plans and Usage, I see the Github Copilot under Add-ons, and there's just no option for me to subscribe monthly for Copilot.  There's an option that says "Sign up for Copilot for Individuals" though.  So I clicked that, and it takes me to the Copilot section in Settings and gives me links to install the Copilot extension for different IDEs, but I already have the Copilot extension installed for Visual Studio Code.
I'm just confused because I simply just want to do a monthly subscription for Copilot but I can't figure out how to do it

**Comment by Victor-Njoroge** [comment]
Explore GitHub Marketplace:

GitHub offers various services and integrations through its Marketplace. It's possible that you might find GitHub Copilot subscription options there.

**Comment by davecheney** [comment, marked as answer]
@datalore92 your 30 day trial expires today (as of writing) no action is required if you wish to continue to use the service, you will be billed monthly from tomorrow.

**Reply by robotnumber3** [reply]
Hi, Dave. I signed up for the 30-day trial, but canceled it after several weeks because I wasn't sure if I wanted to continue using Copilot. It's great, and I want to continue. However, like the previous person said, there is nowhere on the Billing page to actually sign up for the Individual plan. My 30-day trial period ends in a week. Will I still receive email to sign up for the Individual plan? Thanks.

**Reply by davecheney** [reply]
@robotnumber3 you are currently a trial user, at the end of your trial period you will automatically roll over to a paid subscription.

**Reply by robotnumber3** [reply]
@datalore92 Dave, thanks very much for replying. So even though I canceled the trial period, it will still roll over to a paid subscription?

**Reply by davecheney** [reply]
According to the information I have available to me, you have not cancelled your subscription. I recommend contacting support, https://support.github.com/contact.

**Reply by robotnumber3** [reply]
@datalore92 Thank you for your help. I'll contact them.

**Reply by lijma** [reply]
i have the same issue.  github contact didn't solve my request since last week.  any other good recommendation>
